src/service/database/index.ts

```diff
@@ -46,9 +53,48 @@ export class DatabaseService {
 
     for (const row of data) {
       const [name, description] = row as string[];
-      databases.push(new DatabaseModel({ name, description }));
+      databases.push(new DatabaseModel({ name, description }, this.context));
     }
 
     return databases;
   }
+
+  async create(
+    name: string,
+    region: string | undefined = undefined,
+    attached_engines: string[] | EngineModel[] | undefined = undefined,
+    description: string | undefined = undefined,
+    fail_if_exists: boolean = true
+  ): Promise<DatabaseModel> {
+    let query: string =
+      "CREATE DATABASE " + (fail_if_exists ? "" : "IF NOT EXISTS ") + name;
```

Comment:
when doing actions with database or engine we should wrap name in `""`. Plain name without "" will fail when name has number in it.